proto/cosmos/staking/v1beta1/tx.proto

```diff
@@ -71,7 +101,12 @@ message MsgBeginRedelegate {
   cosmos.base.v1beta1.Coin amount                = 4 [(gogoproto.nullable) = false];
 }
 
-// MsgUndelegate defines an SDK message for performing an undelegation from a
+// MsgBeginRedelegateResponse defines the Msg/BeginRedelegate response type.
+message MsgBeginRedelegateResponse { 
+  bytes completion_time = 1;
```

Comment:
Should we use `google.protobuf.Timestamp` instead?

Reply:
Yes, I think so